client/blocks/reader-share/style.scss

```diff
@@ -144,18 +144,18 @@
 		fill: var(--color-primary);
 	}
 
-	.reader-twitter {
-		fill: var(--color-twitter);
```

Comment:
Seems like we are using this variable in a couple of other places which also needs update (for follow up PR)

Reply:
Issue created here: https://github.com/Automattic/wp-calypso/issues/99873